pkgs/development/python-modules/sumtypes/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchFromGitHub, attrs, pytest }:
+
+buildPythonPackage rec {
+  pname = "sumtypes";
+  version = "0.1a5";
+
+  src = fetchFromGitHub {
+    owner = "radix";
+    repo = "sumtypes";
+    rev = version;
+    sha256 = "0wcw1624xxx2h6lliv13b59blg60j8sgf5v2ni3cwx3j4wld4csr";
+  };
+
+  propagatedBuildInputs = [ attrs ];
+  checkInputs = [ pytest ];
```

Comment:
Can you use pytestCheckHook?

Reply:
Ah, so that's why no tests were being ran! Thank you, I pushed a fix.